### PR TITLE
Add copy operator to fix memory benchmarks

### LINKED ISSUE #6529: Incorrect memory reporting in benchmarks
## 🧰 Task

In the benchmarks, the memory usage of layers is wrongly reported. Even if local running asizeof return bigger values. 

The reported memory usage should be bigger than data memory itself. 

```
[80.99%] ··· benchmark_labels_layer.Labels2DSuite.mem_data                   ok
[80.99%] ··· ====== ============= =============
             --                dtype           
             ------ ---------------------------
               n     numpy.uint8   numpy.int32 
             ====== ============= =============
               16        384          1.15k    
               32       1.15k         4.22k    
               64       4.22k         16.5k    
              128       16.5k         65.7k    
              256       65.7k          262k    
              512        262k         1.05M    
              1024      1.05M         4.19M    
              2048      4.19M         16.8M    
              4096      16.8M         67.1M    
             ====== ============= =============

[81.20%] ··· benchmark_labels_layer.Labels2DSuite.mem_layer                  ok
[81.20%] ··· ====== ============= =============
             --                dtype           
             ------ ---------------------------
               n     numpy.uint8   numpy.int32 
             ====== ============= =============
               16       3.29k         3.29k    
               32       3.29k         3.29k    
               64       3.29k         3.29k    
              128       3.29k         3.29k    
              256       3.29k         3.29k    
              512       3.29k         3.29k    
              1024      3.29k         3.29k    
              2048      3.29k         3.29k    
              4096      3.29k         3.29k    
             ====== ============= =============
```

### PR DESCRIPTION
**Pull request by Czaki**
# References and relevant issues
closes #6529

# Description

This PR provides two changes to fix benchmarking of memory usage by layers. 

I have found that `asv` uses following code to calculate memory usage for an object. 

```python
asizeof([obj, copy(obj)]) - asizeof([obj, obj])
```

This approach allows not to count of global (or class) object into size of single object. But if the copy of an object is shallow, then the reported size is small. 

So in this PR, I define `__copy__` method of Base layer, that uses `as_layer_data_tuple` method and `create` class method to create a copy. It also creates copy of data, as it is a significant part of layer size. 

I also found a second problem. 

The `aszieof` function on output of jited version of `_zero_preserving_modulo` reports small values (like 128 bytes). But when I lift array allocation outside implementation, the memory is reported properly. So I have updated the `_zero_preserving_modulo` implementation to allocate memory before entering the jited code. 

Current main:

```
[50.00%] ··· benchmark_labels_layer.Labels2DSuite.mem_data                                                         ok
[50.00%] ··· ====== ============= =============
             --                dtype
             ------ ---------------------------
               n     numpy.uint8   numpy.int32
             ====== ============= =============
               16        384          1.15k
               32       1.15k         4.22k
               64       4.22k         16.5k
              128       16.5k         65.7k
              256       65.7k          262k
              512        262k         1.05M
              1024      1.05M         4.19M
              2048      4.19M         16.8M
              4096      16.8M         67.1M
             ====== ============= =============

[100.00%] ··· benchmark_labels_layer.Labels2DSuite.mem_layer                                                        ok
[100.00%] ··· ====== ============= =============
              --                dtype
              ------ ---------------------------
                n     numpy.uint8   numpy.int32
              ====== ============= =============
                16       3.29k         3.29k
                32       3.29k         3.29k
                64       3.29k         3.29k
               128       3.29k         3.29k
               256       3.29k         3.29k
               512       3.29k         3.29k
               1024      3.29k         3.29k
               2048      3.29k         3.29k
               4096      3.29k         3.29k
              ====== ============= =============
```

with added copy:
```
[50.00%] ··· benchmark_labels_layer.Labels2DSuite.mem_data                                                         ok
[50.00%] ··· ====== ============= =============
             --                dtype
             ------ ---------------------------
               n     numpy.uint8   numpy.int32
             ====== ============= =============
               16        384          1.15k
               32       1.15k         4.22k
               64       4.22k         16.5k
              128       16.5k         65.7k
              256       65.7k          262k
              512        262k         1.05M
              1024      1.05M         4.19M
              2048      4.19M         16.8M
              4096      16.8M         67.1M
             ====== ============= =============

[100.00%] ··· benchmark_labels_layer.Labels2DSuite.mem_layer                                                        ok
[100.00%] ··· ====== ============= =============
              --                dtype
              ------ ---------------------------
                n     numpy.uint8   numpy.int32
              ====== ============= =============
                16        188k          189k
                32        189k          192k
                64        192k          205k
               128        205k          254k
               256        254k          450k
               512        450k         1.24M
               1024      1.24M         4.38M
               2048      4.38M          17M
               4096       17M          67.3M
              ====== ============= =============
```

with copy and refactored `_zero_preserving_modulo`
```
[50.00%] ··· benchmark_labels_layer.Labels2DSuite.mem_data                                                         ok
[50.00%] ··· ====== ============= =============
             --                dtype
             ------ ---------------------------
               n     numpy.uint8   numpy.int32
             ====== ============= =============
               16        384          1.15k
               32       1.15k         4.22k
               64       4.22k         16.5k
              128       16.5k         65.7k
              256       65.7k          262k
              512        262k         1.05M
              1024      1.05M         4.19M
              2048      4.19M         16.8M
              4096      16.8M         67.1M
             ====== ============= =============

[100.00%] ··· benchmark_labels_layer.Labels2DSuite.mem_layer                                                        ok
[100.00%] ··· ====== ============= =============
              --                dtype
              ------ ---------------------------
                n     numpy.uint8   numpy.int32
              ====== ============= =============
                16        188k          189k
                32        189k          193k
                64        192k          209k
               128        205k          270k
               256        254k          516k
               512        450k          1.5M
               1024      1.24M         5.43M
               2048      4.38M         21.2M
               4096       17M          84.1M
              ====== ============= =============
```

### CODE REVIEW

**Comment by brisvag** [comment]
Shouldn't this be the `deepcopy` behaviour then?

**Review by jni — APPROVED** [review]

**Comment by jni** [comment]
I'm not super concerned about exactly following the copy/deepcopy semantics. This is mostly for internal usage I think...

**Comment by jni** [comment]
(But if you have specific suggestions @brisvag that are easy to implement here I of course wouldn't be opposed to including them!)

**Comment by brisvag** [comment]
If it's just for internal use which is deviating from the standard of copy/deepcopy, then I would suggest making a private method for it, like `_copy_with_data`... as a user I would expect `copy(layer)` and `deepcopy(layer)` to follow the nodata/data intuition.

**Comment by Czaki** [comment]
I have found that `asizeof` wrongly calculated the size of array allocated in jited function

File: napari/layers/base/base.py [hunk 1]
```diff
@@ -2058,6 +2059,17 @@ def save(self, path: str, plugin: Optional[str] = None) -> List[str]:
...
 
+    def __copy__(self):
+        """Create a copy of this layer.
+
+        Returns
+        -------
+        layer : napari.layers.Layer
+            Copy of this layer.
+        """
+        data, meta, layer_type = self.as_layer_data_tuple()
```
Comment (brisvag) [inline napari/layers/base/base.py:2082]: I would then add a comment here saying that benchmarks will break if we change the copy behaviour.

**Review by brisvag — APPROVED** [review]
LGTM, thanks for adding comments and a detailed description!

**Comment by Czaki** [comment]
Benchmarks failing as expected. :)

(correct layer size is much bigger than the previous one)

```

| Change   | Before [3b52a100]    | After [981f8207]    |   Ratio | Benchmark (Parameter)                                                                  |
|----------|----------------------|---------------------|---------|----------------------------------------------------------------------------------------|
| -        | 96.2±1ms             | 177±3μs             |       0 | benchmark_labels_layer.Labels2DColorDirectSuite.time_fill(4096, <class 'numpy.uint8'>) |
| Change   | Before [3b52a100]    | After [981f8207]    |     Ratio | Benchmark (Parameter)                                                                  |
|----------|----------------------|---------------------|-----------|----------------------------------------------------------------------------------------|
| +        | 1.64k                | 164k                |     99.97 | benchmark_surface_layer.Surface2DSuite.mem_layer(64)                                   |
| +        | 1.64k                | 161k                |     98.47 | benchmark_surface_layer.Surface3DSuite.mem_layer(16)                                   |
| +        | 3.29k                | 323k                |     98.28 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(128, <class 'numpy.int32'>)  |
| +        | 1.64k                | 161k                |     98.09 | benchmark_surface_layer.Surface2DSuite.mem_layer(16)                                   |
| +        | 1.54k                | 151k                |     98.01 | benchmark_vectors_layer.Vectors2DSuite.mem_layer(16)                                   |
| +        | 1.54k                | 151k                |     97.69 | benchmark_vectors_layer.Vectors3DSuite.mem_layer(16)                                   |
| +        | 3.29k                | 320k                |     97.44 | benchmark_labels_layer.Labels3DSuite.mem_layer(32, <class 'numpy.uint32'>)             |
| +        | 1.64k                | 153k                |     93.06 | benchmark_image_layer.Image2DSuite.mem_layer(32)                                       |
| +        | 1.64k                | 146k                |     89.32 | benchmark_image_layer.Image2DSuite.mem_layer(16)                                       |
| +        | 3.29k                | 273k                |     83.14 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(128, <class 'numpy.uint8'>)  |
| +        | 3.29k                | 270k                |     82.15 | benchmark_labels_layer.Labels2DSuite.mem_layer(128, <class 'numpy.int32'>)             |
| +        | 1.64k                | 134M                |  81928.1  | benchmark_image_layer.Image2DSuite.mem_layer(4096)                                     |
| +        | 1.64k                | 134M                |  81928.1  | benchmark_image_layer.Image3DSuite.mem_layer(256)                                      |
| +        | 3.29k                | 254k                |     77.16 | benchmark_labels_layer.Labels2DSuite.mem_layer(256, <class 'numpy.uint8'>)             |
| +        | 1.64k                | 1.21M               |    736.84 | benchmark_surface_layer.Surface2DSuite.mem_layer(16384)                                |
| +        | 1.54k                | 1.12M               |    727.07 | benchmark_shapes_layer.Shapes2DSuite.mem_layer(128)                                    |
| +        | 3.29k                | 2.29M               |    696.24 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(512, <class 'numpy.int32'>)  |
| +        | 3.29k                | 2.29M               |    695.09 | benchmark_labels_layer.Labels3DSuite.mem_layer(128, <class 'numpy.uint8'>)             |
| +        | 3.29k                | 225k                |     68.38 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(64, <class 'numpy.int32'>)   |
| +        | 3.29k                | 221k                |     67.23 | benchmark_labels_layer.Labels3DSuite.mem_layer(32, <class 'numpy.uint8'>)              |
| +        | 1.64k                | 1.07G               | 654809    | benchmark_image_layer.Image3DSuite.mem_layer(512)                                      |
| +        | 3.29k                | 21.2M               |   6436.62 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(2048, <class 'numpy.uint8'>) |
| +        | 3.29k                | 21.2M               |   6435.43 | benchmark_labels_layer.Labels2DSuite.mem_layer(2048, <class 'numpy.int32'>)            |
| +        | 3.29k                | 211k                |     64.25 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(64, <class 'numpy.uint8'>)   |
| +        | 3.29k                | 209k                |     63.46 | benchmark_labels_layer.Labels2DSuite.mem_layer(64, <class 'numpy.int32'>)              |
| +        | 3.29k                | 205k                |     62.33 | benchmark_labels_layer.Labels3DSuite.mem_layer(16, <class 'numpy.uint32'>)             |
| +        | 3.29k                | 205k                |     62.21 | benchmark_labels_layer.Labels2DSuite.mem_layer(128, <class 'numpy.uint8'>)             |
| +        | 1.54k                | 941k                |    609.29 | benchmark_shapes_layer.Shapes3DSuite.mem_layer(128)                                    |
| +        | 3.29k                | 200k                |     60.91 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(32, <class 'numpy.int32'>)   |
| +        | 3.29k                | 197k                |     59.98 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(32, <class 'numpy.uint8'>)   |
| +        | 3.29k                | 194k                |     59.04 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(16, <class 'numpy.int32'>)   |
| +        | 3.29k                | 193k                |     58.79 | benchmark_labels_layer.Labels2DSuite.mem_layer(32, <class 'numpy.int32'>)              |
| +        | 3.29k                | 193k                |     58.61 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(16, <class 'numpy.uint8'>)   |
| +        | 3.29k                | 192k                |     58.51 | benchmark_labels_layer.Labels3DSuite.mem_layer(16, <class 'numpy.uint8'>)              |
| +        | 3.29k                | 192k                |     58.48 | benchmark_labels_layer.Labels2DSuite.mem_layer(64, <class 'numpy.uint8'>)              |
| +        | 3.29k                | 189k                |     57.62 | benchmark_labels_layer.Labels2DSuite.mem_layer(16, <class 'numpy.int32'>)              |
| +        | 3.29k                | 189k                |     57.54 | benchmark_labels_layer.Labels2DSuite.mem_layer(32, <class 'numpy.uint8'>)              |
| +        | 3.29k                | 188k                |     57.31 | benchmark_labels_layer.Labels2DSuite.mem_layer(16, <class 'numpy.uint8'>)              |
| +        | 1.64k                | 8.53M               |   5203.07 | benchmark_image_layer.Image2DSuite.mem_layer(1024)                                     |
| +        | 3.29k                | 17M                 |   5159.83 | benchmark_labels_layer.Labels3DSuite.mem_layer(256, <class 'numpy.uint8'>)             |
| +        | 3.29k                | 17M                 |   5159.79 | benchmark_labels_layer.Labels2DSuite.mem_layer(4096, <class 'numpy.uint8'>)            |
| +        | 1.54k                | 7.88M               |   5101.23 | benchmark_points_layer.Points3DSuite.mem_layer(65536)                                  |
| +        | 1.54k                | 7.62M               |   4932.9  | benchmark_points_layer.Points2DSuite.mem_layer(65536)                                  |
| +        | 3.29k                | 1.5M                |    457.06 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(512, <class 'numpy.uint8'>)  |
| +        | 3.29k                | 1.5M                |    455.87 | benchmark_labels_layer.Labels2DSuite.mem_layer(512, <class 'numpy.int32'>)             |
| +        | 1.54k                | 6.97M               |   4511.27 | benchmark_vectors_layer.Vectors2DSuite.mem_layer(65536)                                |
| +        | 1.54k                | 690k                |    446.78 | benchmark_points_layer.Points3DSuite.mem_layer(4096)                                   |
| +        | 1.54k                | 674k                |    436.32 | benchmark_points_layer.Points2DSuite.mem_layer(4096)                                   |
| +        | 1.54k                | 6.7M                |   4339.69 | benchmark_vectors_layer.Vectors3DSuite.mem_layer(65536)                                |
| +        | 1.54k                | 648k                |    419.49 | benchmark_shapes_layer.Shapes2DSuite.mem_layer(64)                                     |
| +        | 3.29k                | 134M                |  40878.9  | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(4096, <class 'numpy.int32'>) |
| +        | 3.29k                | 134M                |  40877.7  | benchmark_labels_layer.Labels3DSuite.mem_layer(512, <class 'numpy.uint8'>)             |
| +        | 1.64k                | 669k                |    407.76 | benchmark_image_layer.Image2DSuite.mem_layer(256)                                      |
| +        | 1.64k                | 6.45M               |   3933.79 | benchmark_surface_layer.Surface3DSuite.mem_layer(65536)                                |
| +        | 3.29k                | 1.24M               |    377.42 | benchmark_labels_layer.Labels3DSuite.mem_layer(64, <class 'numpy.uint32'>)             |
| +        | 3.29k                | 1.24M               |    376.14 | benchmark_labels_layer.Labels2DSuite.mem_layer(1024, <class 'numpy.uint8'>)            |
| +        | 1.54k                | 576k                |    372.83 | benchmark_vectors_layer.Vectors2DSuite.mem_layer(4096)                                 |
| +        | 1.54k                | 559k                |    362.1  | benchmark_vectors_layer.Vectors3DSuite.mem_layer(4096)                                 |
| +        | 1.54k                | 555k                |    359.48 | benchmark_shapes_layer.Shapes3DSuite.mem_layer(64)                                     |
| +        | 1.64k                | 553k                |    337.3  | benchmark_surface_layer.Surface3DSuite.mem_layer(4096)                                 |
| +        | 1.64k                | 4.35M               |   2654.97 | benchmark_surface_layer.Surface2DSuite.mem_layer(65536)                                |
| +        | 1.54k                | 410k                |    265.5  | benchmark_shapes_layer.Shapes2DSuite.mem_layer(32)                                     |
| +        | 3.29k                | 8.59M               |   2613.53 | benchmark_labels_layer.Labels3DSuite.mem_layer(128, <class 'numpy.uint32'>)            |
| +        | 3.29k                | 8.58M               |   2609.7  | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(1024, <class 'numpy.int32'>) |
| +        | 1.64k                | 422k                |    257.31 | benchmark_surface_layer.Surface2DSuite.mem_layer(4096)                                 |
| +        | 3.29k                | 84.1M               |  25571.2  | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(4096, <class 'numpy.uint8'>) |
| +        | 3.29k                | 84.1M               |  25570    | benchmark_labels_layer.Labels2DSuite.mem_layer(4096, <class 'numpy.int32'>)            |
| +        | 1.64k                | 406k                |    247.85 | benchmark_image_layer.Image3DSuite.mem_layer(32)                                       |
| +        | 1.54k                | 365k                |    236.2  | benchmark_shapes_layer.Shapes3DSuite.mem_layer(32)                                     |
| +        | 3.29k                | 716k                |    217.87 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(256, <class 'numpy.int32'>)  |
| +        | 1.54k                | 330k                |    213.85 | benchmark_points_layer.Points3DSuite.mem_layer(1024)                                   |
| +        | 1.54k                | 327k                |    211.49 | benchmark_points_layer.Points2DSuite.mem_layer(1024)                                   |
| +        | 1.64k                | 33.7M               |  20548.1  | benchmark_image_layer.Image2DSuite.mem_layer(2048)                                     |
| +        | 3.29k                | 67.4M               |  20487.4  | benchmark_labels_layer.Labels3DSuite.mem_layer(256, <class 'numpy.uint32'>)            |
| +        | 1.54k                | 291k                |    188.17 | benchmark_shapes_layer.Shapes2DSuite.mem_layer(16)                                     |
| +        | 1.54k                | 269k                |    174.51 | benchmark_shapes_layer.Shapes3DSuite.mem_layer(16)                                     |
| +        | 1.64k                | 276k                |    167.99 | benchmark_image_layer.Image2DSuite.mem_layer(128)                                      |
| +        | 3.29k                | 5.43M               |   1652.77 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(1024, <class 'numpy.uint8'>) |
| +        | 3.29k                | 5.43M               |   1651.78 | benchmark_labels_layer.Labels2DSuite.mem_layer(1024, <class 'numpy.int32'>)            |
| +        | 1.54k                | 256k                |    165.91 | benchmark_vectors_layer.Vectors2DSuite.mem_layer(1024)                                 |
| +        | 3.29k                | 537M                | 163419    | benchmark_labels_layer.Labels3DSuite.mem_layer(512, <class 'numpy.uint32'>)            |
| +        | 1.54k                | 253k                |    163.63 | benchmark_vectors_layer.Vectors3DSuite.mem_layer(1024)                                 |
| +        | 3.29k                | 520k                |    158.08 | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(256, <class 'numpy.uint8'>)  |
| +        | 1.64k                | 258k                |    157.48 | benchmark_surface_layer.Surface3DSuite.mem_layer(1024)                                 |
| +        | 3.29k                | 516k                |    156.89 | benchmark_labels_layer.Labels2DSuite.mem_layer(256, <class 'numpy.int32'>)             |
| +        | 1.54k                | 240k                |    155.73 | benchmark_points_layer.Points3DSuite.mem_layer(256)                                    |
| +        | 1.54k                | 240k                |    155.28 | benchmark_points_layer.Points2DSuite.mem_layer(256)                                    |
| +        | 1.54k                | 218k                |    141.23 | benchmark_points_layer.Points2DSuite.mem_layer(64)                                     |
| +        | 1.54k                | 218k                |    141.11 | benchmark_points_layer.Points3DSuite.mem_layer(64)                                     |
| +        | 1.54k                | 2.13M               |   1377.63 | benchmark_points_layer.Points3DSuite.mem_layer(16384)                                  |
| +        | 1.54k                | 213k                |    137.72 | benchmark_points_layer.Points2DSuite.mem_layer(16)                                     |
| +        | 1.54k                | 212k                |    137.47 | benchmark_points_layer.Points3DSuite.mem_layer(16)                                     |
| +        | 1.64k                | 225k                |    137.43 | benchmark_surface_layer.Surface2DSuite.mem_layer(1024)                                 |
| +        | 3.29k                | 450k                |    137    | benchmark_labels_layer.Labels3DSuite.mem_layer(64, <class 'numpy.uint8'>)              |
| +        | 1.64k                | 2.24M               |   1366.82 | benchmark_image_layer.Image2DSuite.mem_layer(512)                                      |
| +        | 1.64k                | 2.24M               |   1366.76 | benchmark_image_layer.Image3DSuite.mem_layer(64)                                       |
| +        | 3.29k                | 450k                |    136.96 | benchmark_labels_layer.Labels2DSuite.mem_layer(512, <class 'numpy.uint8'>)             |
| +        | 1.54k                | 2.09M               |   1351.99 | benchmark_shapes_layer.Shapes2DSuite.mem_layer(256)                                    |
| +        | 1.54k                | 2.06M               |   1335.64 | benchmark_points_layer.Points2DSuite.mem_layer(16384)                                  |
| +        | 3.29k                | 4.38M               |   1332.87 | benchmark_labels_layer.Labels2DSuite.mem_layer(2048, <class 'numpy.uint8'>)            |
| +        | 1.54k                | 1.85M               |   1200.52 | benchmark_vectors_layer.Vectors2DSuite.mem_layer(16384)                                |
| +        | 1.54k                | 1.79M               |   1157.54 | benchmark_vectors_layer.Vectors3DSuite.mem_layer(16384)                                |
| +        | 1.54k                | 176k                |    114.18 | benchmark_vectors_layer.Vectors2DSuite.mem_layer(256)                                  |
| +        | 1.54k                | 175k                |    113.37 | benchmark_vectors_layer.Vectors3DSuite.mem_layer(256)                                  |
| +        | 1.64k                | 185k                |    112.52 | benchmark_surface_layer.Surface3DSuite.mem_layer(256)                                  |
| +        | 1.54k                | 1.72M               |   1112.93 | benchmark_shapes_layer.Shapes3DSuite.mem_layer(256)                                    |
| +        | 1.64k                | 177k                |    108.05 | benchmark_image_layer.Image2DSuite.mem_layer(64)                                       |
| +        | 1.64k                | 177k                |    107.99 | benchmark_image_layer.Image3DSuite.mem_layer(16)                                       |
| +        | 1.64k                | 176k                |    107.46 | benchmark_surface_layer.Surface2DSuite.mem_layer(256)                                  |
| +        | 1.64k                | 1.73M               |   1056.6  | benchmark_surface_layer.Surface3DSuite.mem_layer(16384)                                |
| +        | 1.64k                | 16.9M               |  10318    | benchmark_image_layer.Image3DSuite.mem_layer(128)                                      |
| +        | 3.29k                | 33.7M               |  10263.5  | benchmark_labels_layer.Labels2DColorDirectSuite.mem_layer(2048, <class 'numpy.int32'>) |
| +        | 1.64k                | 166k                |    101.28 | benchmark_surface_layer.Surface3DSuite.mem_layer(64)                                   |
| +        | 1.54k                | 156k                |    101.24 | benchmark_vectors_layer.Vectors2DSuite.mem_layer(64)                                   |
| +        | 1.54k                | 155k                |    100.7  | benchmark_vectors_layer.Vectors3DSuite.mem_layer(64)                                   |
```

**Pull request merged**